docs/source/tutorials/debugging.ipynb

```diff
@@ -6,13 +6,13 @@
    "source": [
     "# Debugging\n",
     "\n",
-    "RADICAL-Pilot is a complex runtime system which employes multiple distributed components to orchestrate workload execution.  It is also a *research software*,  funded by research grants.  As such it is possibly it is not quite comparable to commercially supported software systems.  \n",
+    "RADICAL-Pilot is a complex runtime system which employs multiple distributed components to orchestrate workload execution.  It is also a **_research software_**,  funded by research grants.  As such it is possibly it is not quite comparable to commercially supported software systems.  \n",
```

Comment:
`As such it is possibly it is not quite comparable to commercially supported software systems.` --> `As such, it is possibly not comparable to commercially supported software systems.`

Reply:
fixed